src/main/java/crawlercommons/sitemaps/sax/RSSHandler.java

```diff
@@ -109,47 +107,55 @@ public void startElement(String uri, String localName, String qName, Attributes
      */
     @Override
     public void endElement(String uri, String localName, String qName) throws SAXException {
-        if ("link".equals(currentElement()) || "guid".equals(currentElement())) {
+        if ("link".equals(localName)) {
```

Comment:
This looks like a separate fix for some issue (vs. what's in the CHANGES.txt entry), yes? What was the `guid` check for previously?

Reply:
OK, wait - I think I see what's changing in the logic (`guid` check was in `characters()`).

Reply:
Yes, `guid` was before completely handled in `characters()`, now the content is aggregated in `characters()` and is consumed in `endElement()`.